R/utils_dataprocess_plots.R

```diff
@@ -50,29 +49,34 @@
         input$is_censored = factor(input$is_censored, 
                                    levels = c("FALSE", "TRUE"))
     }
+    print(input)
+    print(unique(input$predicted))
+    write.csv(input, file = "your_file_name.csv", row.names = T)
     featureName = toupper(featureName)
     if (featureName == "TRANSITION") {
         type_color = "FEATURE"
     } else {
         type_color = "PEPTIDE"
     }
     
-    profile_plot = ggplot(input, aes_string(x = 'RUN', y = 'ABUNDANCE', 
+    profile_plot = ggplot(input, aes_string(x = 'RUN', y = 'newABUNDANCE',
                                             color = type_color, linetype = 'FEATURE')) +
         facet_grid(~LABEL) +
         geom_line(size = 0.5)
     
     if (is_censored) {
         profile_plot = profile_plot +
-            geom_point(aes_string(x = 'RUN', y = 'ABUNDANCE', color = type_color, shape = 'censored'), 
-                       data = input, 
-                       size = dot.size.profile) +
-            scale_shape_manual(values = c(16, 1), 
-                               labels = c("Detected data", "Censored missing data"))
+        geom_point(aes_string(x = 'RUN', y = 'newABUNDANCE', color = type_color, shape = 'censored'),
+                   data = input,
+                   size = dot.size.profile) +
+        scale_shape_manual(values = c(16, 1),
+                           labels = c("Detected data", "Censored missing data"))
+        print("IS CENS")
```

Comment:
where will this be printed?

Reply:
Commit [remove debug statements](https://github.com/Vitek-Lab/MSstats/pull/107/commits/518b8f9c932d7fd5ca86ee85f0bab533da0530c0)